identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```
@@ -611,3 +611,7 @@ error.duo.user.not.found=Couldn't get the verified user name.
 error.mobile.not.found=User may not have a mobile number in user's profile for Duo Authentication. Please update it.
 error.mobile.not.found.duo=Unable to get the registered phone number from Duo. Authentication failed.
 error.number.mismatch.duo=Authentication failed due to mismatch in mobile numbers. Please update the correct registered duo mobile number in user profile.
+
+# Conditional authentication scripts
+prompt.template.internal.wait.greeting=Just a moment..
```

Ellipsis should have 3 dots. Let's update the other places as well accordingly

```suggestion
prompt.template.internal.wait.greeting=Just a moment...
```

---

Updated